### ISSUE TITLE
libhamlib2 has been retired, any chance of an update to support libhamlib4

### ISSUE THREAD

**Issue by daniel-tickell** [tzneal/ham-go#17]
Hi, 

on Ubuntu bullseye the libhamlib2 library isnt available in the repo, is there any chance of an update to your software to support libhamlib4

Thanks

Daniel

**Comment by benishor** [comment]
Having the same problem. The problem seems to be related to github.com/dh1tw/goHamlib not to ham-go itself. Not sure what to do next other than try to fork and update goHamlib

**Comment by dl1igc** [comment]
Check here https://github.com/dh1tw/goHamlib/pull/14

**Comment by dh1tw** [comment]
I agree.. we should really upgrade to a more modern version of hamlib.
